product_auction/views/product_grade_views.xml

```diff
@@ -5,7 +5,8 @@
         <field name="model">product.grade</field>
         <field name="type">tree</field>
         <field name="arch" type="xml">
-            <tree create="1">
+            <tree create="1" default_order="sequence">
```

Comment:
```suggestion
            <tree create="1">
```
I think it's redundant to set default_order in the view when it's set in the model.